pool/src/storage.rs

```diff
@@ -347,9 +347,7 @@ pub fn set_pool_config(e: &Env, config: &PoolConfig) {
 /// If the reserve does not exist
 pub fn get_res_config(e: &Env, asset: &Address) -> ReserveConfig {
     let key = PoolDataKey::ResConfig(asset.clone());
-    e.storage()
-        .persistent()
-        .extend_ttl(&key, LEDGER_THRESHOLD_SHARED, LEDGER_BUMP_SHARED);
+    e.storage().persistent().extend_ttl(&key, 0, 0);
```

Comment:
was this a typo / test additional that did not get reverted?

Reply:
Yeah forgot to revert from my ledger restore pr. This has been fixed